docs/source/routing/observability/telemetry.mdx

```diff
@@ -20,139 +31,300 @@ flowchart LR
 ```
 
-### Logs and events
-
-Logs record **events** in the router. Examples of logged events include:
+### Metrics and instruments
 
-* Information about the router lifecycle
-* Warnings about misconfiguration
-* Errors that occurred during a request
+Metrics are measurements of the router's behavior, for example, the number of incoming HTTP requests. Metrics are numeric values that are collected and often analyzed over time to identify trends.
+
+Instruments define _how_ to collect and report metrics. Different kinds of instruments include counters, gauges, and histograms. Given the metric “number of incoming HTTP requests”, different instruments collect a count, histogram, and gauge of this metric.
+
+#### Instrument types
+
+Metric instruments fall into three categories:
+
+<table>
+  <thead>
+    <tr>
+      <th>Instrument Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><strong>Router instruments</strong></td>
+      <td>
+        Standard instruments for the router's request life cycle, including:
+        <ul>
+          <li>The number of HTTP requests by HTTP status</li>
+          <li>A histogram of HTTP router request duration</li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td><strong>OTEL instruments</strong></td>
+      <td>
+        Standard OpenTelemetry instruments around the HTTP lifecycle, including:
+        <ul>
+          <li>The number of active requests in flight</li>
+          <li>A histogram of request body sizes</li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td><strong>Custom instrument</strong></td>
+      <td>
+        Custom instruments defined in the router's request life cycle.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+#### Example instrument configuration
+
+This configuration snippet enables OTEL instrumentation for a histogram of request body sizes:
+
+```yaml title="router.yaml"
+telemetry:
+  instrumentation:
+    instruments:
+      router:
+        http.server.request.body.size: true
+```
 
-Logs can be consumed by [logging exporters](/router/configuration/telemetry/exporters/logging/overview) and as part of [spans](#traces-and-spans) via [tracing exporters](/router/configuration/telemetry/exporters/tracing/overview). 
+See [Instruments](/router/configuration/telemetry/instrumentation/instruments) for an overview of available instruments and a guide for configuring and customizing instruments.
 
-### Metrics and instruments
+#### Metric exporters
 
-Metrics are measurements of the router's behavior that can be exported and monitored. Different kinds of metrics include histograms, gauges, and counts. 
+The router sends [operation](/graphos/platform/insights/sending-operation-metrics#from-the-apollo-router-or-apollo-server) and [field usage metrics](/graphos/platform/insights/sending-operation-metrics#from-the-apollo-router) metrics to GraphOS. Additionally, metric exporters can consume and monitor other metrics.
```

Comment:
Need to distinguish the special case of operation metrics reported to GraphOS, as they're not one of the exporters configurable by `telemetry.exporters.*`

```suggestion
In addition to the [operation metrics](/graphos/platform/insights/sending-operation-metrics#from-the-apollo-router-or-apollo-server) and [field usage metrics](/graphos/platform/insights/sending-operation-metrics#from-the-apollo-router) that GraphOS Router sends to GraphOS, you can configure the router with metric exporters for other observability tools and APMs.
```